auditbeat/module/auditd/_meta/kibana/7/dashboard/auditbeat-kernel-executions.json

```diff
@@ -12,18 +12,18 @@
                         }
                     }
                 }, 
-                "savedSearchId": "d382f5b0-c1c6-11e7-8995-936807a28b16", 
-                "title": "Error Codes [Auditbeat Auditd Executions]", 
+                "savedSearchId": "d382f5b0-c1c6-11e7-8995-936807a28b16-ecs", 
+                "title": "Error Codes [Auditbeat Auditd Executions] ECS", 
                 "uiStateJSON": {}, 
-                "version": 1, 
+                "winlog.version": 1, 
```

Comment:
@webmat Looks like your winlogbeat changes break my script :-(

Reply:
How so?

Reply:
Oh, got it. Isn't my ecs-migration.yml entry scoped to winlogbeat?

Reply:
For now I skipped version and type to be renamed as they were not used in the dashboards.